org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/thingTranslator.js

```diff
@@ -79,7 +79,7 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
         entities: [
             {
                 n: providerSchema['name'],
-                ivc: "core.1.1.0",
+                icv: "core.1.1.0",
```

Comment:
We never tested it, and we don't have multiple core versions at the moment, so I doubt anybody looks at this value today.